### PR TITLE
Add OracleChain API calls

### LINKED ISSUE #72: Provide functions get oracle data
### Is your feature request related to a problem?

The node is exposing a GraphQL api queries and subscriptions to get the last oracle data
The libjs should have a set of functions to leverage it.

### Describe the solution you'd like

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by samuelmanzanera**
Provide new functions to support the new OracleChain API

Fixes #72

### CODE REVIEW

**Review by prix-uniris — COMMENTED** [review]
LGTM, Tested working well in nodejs env

**Review by prix-uniris — APPROVED** [review]

File: lib/api.js [hunk 1]
```diff
@@ -112,7 +112,111 @@ module.exports = {
       })
     })
       .then(handleResponse)
-  }
+  },
+  
+  getLastOracleData: function (endpoint) {
```
Comment (apoorv-2204) [inline lib/api.js:117]: I think we should also have mock,unit test for js lib

**Comment by apoorv-2204** [comment]
Screenshot_20220905_162507
great :+1:

File: lib/api.js (hunk 1 above)
Comment (samuelmanzanera) [reply lib/api.js:117]: While it's interesting, we should add another issue to improve testing, because all the API calls tests have to be implemented

Comment (samuelmanzanera) [reply lib/api.js:117]: #80

**Comment by samuelmanzanera** [comment]
> ```
> const withAbsintheSocket = require("@absinthe/socket")
> const { Socket } = require("phoenix")
> const WebSocket = require("isomorphic-ws")
> ```
> 
> imports?

Why not.
You can do a PR ;)

**Review by apoorv-2204 — APPROVED** [review]
:+1:

**Review by apoorv-2204 — APPROVED** [review]
Great :+1:

**Pull request merged**